go/vt/vtctl/query.go

```diff
@@ -167,6 +160,10 @@ func parseExecuteOptions(value string) (*querypb.ExecuteOptions, error) {
 }
 
 func commandVtGateExecute(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag.FlagSet, args []string) error {
+	if !*enableQueries {
+		return fmt.Errorf("query commands are disabled")
```

Comment:
Maybe include in the error message something like:
Add the -enable_queries flag to enable them
?

Reply:
@alainjobart good suggestion -- updated (and I fixed a misleading comment along the way)